### ISSUE TITLE
False positive SA0102

### ISSUE THREAD

**Issue by Dutchdre** [StyleCop/StyleCop#129]
The following properties in my viewmodel are causing a false positive SA0102:
```c#
public string Copyright
{
	get
	{
		return FileVersionInfo.GetVersionInfo(Assembly.GetExecutingAssembly‌().Location).LegalCopyright;
	}
}

public string Version
{
	get 
	{ 
		return FileVersionInfo.GetVersionInfo(Assembly.GetExecutingAssembly‌().Location).FileVersion; 
	}
}
```

The following fix resolves the false positive:
```c#
public string Copyright
{
	get
	{
		var assembly = Assembly.GetExecutingAssembly().Location;
		return FileVersionInfo.GetVersionInfo(assembly).LegalCopyright;
	}
}
		
public string Version
{
	get
	{
		var assembly = Assembly.GetExecutingAssembly().Location;
		return FileVersionInfo.GetVersionInfo(assembly).FileVersion;
	}
}
```

**Comment by SanjayGuntur** [comment]
I tried to repro this issue and couldn't. Just to check if I missed something, I copied the provided code and I could see the issue. 
It appears that there is some strange character between GetExecutingAssembly and the following open parenthesis. Could you test again and confirm that this is the case?

**Comment by Dutchdre** [comment]
That's right! Awesome stack overflow copy-paste error. Sorry about that.
